storage/http_client.go

```diff
@@ -885,7 +885,7 @@ func (c *httpStorageClient) OpenWriter(params *openWriterParams, opts ...storage
 	mediaOpts := []googleapi.MediaOption{
 		googleapi.ChunkSize(params.chunkSize),
 	}
-	if c := attrs.ContentType; c != "" {
+	if c := attrs.ContentType; c != "" || params.forceEmptyContentType {
```

Comment:
This looks good, however there's a second implementation of OpenWriter for gRPC that you need to implement this in as well. Fortunately it looks like an easy fix here: https://github.com/googleapis/google-cloud-go/blob/cc83515d0c837c8b1596a97b6f09d519a0f75f72/storage/grpc_client.go#L1784

The integration test currently fails for gRPC; it should pass once this is resolved.

Reply:
Ah, thanks. When I ran the integration test, I just assumed the gRPC failure was something else for some reason. Glad that it caught this.

Reply:
I was looking for the output of the integration tests, but I didn't actually see them anywhere in the checks. Are they only accessible to Google team members?

Reply:
Unfortunately they are only run as post-submits by the CI, but I ran locally to double check :)